docs/HOWTO/Custom_Layers_Guide.md

```diff
@@ -1,200 +1,362 @@
-# Custom Layers Guide {#openvino_docs_HOWTO_Custom_Layers_Guide}
+# Custom Operations Guide {#openvino_docs_HOWTO_Custom_Layers_Guide}
+
```

Comment:
Looks like this document and https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_DG/Extensibility_DG/Intro.md are about the same. Do we need this one? 

I suppose we need to **merge** them at least.

Reply:
This guide covers the whole pipeline of the operation enabling including MO and IE. The https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_DG/Extensibility_DG/Intro.md is about extending IE part only. There was a specific request from marketing to have a guide which covers the whole story of operation enabling in one place, so I cannot merge the guides.